packages/web/src/common/store/user-list/reducer.ts

```diff
@@ -6,13 +6,13 @@ import { UserListStoreState } from './types'
 type UserListActions = ActionType<typeof actions>
 
 export const UserListReducerFactory = {
-  createReducer: (tag: string) => {
+  createReducer: (tag: string, pageSize: number) => {
```

Comment:
```suggestion
  createReducer: (tag: string, pageSize: number = 1) => {
```

And then you can do just `pageSize` below

Also it seems like maybe the default should be 15?